src/fragments/lib/project-setup/ios/upgrade-guide/upgrade-guide.mdx

```diff
@@ -0,0 +1,76 @@
+As of v2, Amplify Library iOS is now Amplify Library for Swift. Amplify Library for Swift supports iOS 13+ and macOS 10.15+, and ships with APIs that leverage Swift Concurrency (async/await) to return values. There are a few things you need to know if you are upgrading to this latest version.
+
+## What's changed
+
+We have re-written our APIs to support idiomatic Swift features including [structured concurrency](#structured-concurrency-asyncawait-pattern) and [combine support](/lib/project-setup/combine). We have also changed [how you access the underlying SDK](#escape-hatches) (keep in mind that the AWS SDK for Swift is still in Developer Preview). 
```

Comment:
This link is broken. Please ensure it points to */lib/project-setup/combine/q/platform/ios/*

Reply:
This seems to work for me. As per @timngyn [suggestion](https://github.com/aws-amplify/docs/pull/4759#discussion_r1011270846) we do not need to give the extra platform info in the link.